h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -299,7 +304,13 @@ public EasyPredictModelWrapper(Config config) {
     } else {
       predictContributions = null;
     }
-    
+
+    if ((m.getCategoricalEncoding() != CategoricalEncoding.AUTO) && !config.useExternalEncoding) {
+      throw new UnsupportedOperationException("Categorical Encoding `" + m.getCategoricalEncoding() + 
```

Comment:
Would this be a good candidate for a technical note ?

Reply:
could be, there is no technote infrastructure in genmodel though - I will make a reference to a jira on github where the functionality will be added